src/main/resources/createJsonFilesLambda/createJsonFilesLambda.js

```diff
@@ -98,57 +95,142 @@ async function handleCreate(event, context) {
+ * @param {Context} context the lambda context
+ * @param {string} bucket the S3 bucket
+ * @param {string[]} objects the objects to delete
+ * @returns {Promise<unknown>} the invocation result
+ */
+async function invokeDeleteLambda(context, bucket, objects) {
+    /** @typedef {DeleteListEvent} */
+    const callParams = {
+        action: ACTION_DELETE_LIST,
+        bucket,
+        objects
+    };
+    console.log(`Calling lambda to delete ${objects.length} objects`);
+    const command = new InvokeCommand({
+        FunctionName: context.functionName,
+        Payload: new TextEncoder().encode(JSON.stringify(callParams)),
+        LogType: LogType.Tail
+    });
+    return lambdaClient.send(command);
+}
+
+/**
+ * Split the given array into chunks.
+ * @param {string[]} array the array to split
+ * @param {number} chunkSize the chunk size
+ * @returns {string[][]} the chunks
+ */
+function splitIntoChunks(array, chunkSize) {
+    const result = [];
+    for (let i = 0; i < array.length; i += chunkSize) {
+        const chunk = array.slice(i, i + chunkSize);
+        result.push(chunk);
+    }
+    return result;
+}
+
+/**
+ * Fetch all S3 objects from a bucket.
+ * @param {string} bucket the S3 bucket
+ * @returns {Promise<string[]>} the keys of all objects
+ */
+async function fetchObjects(bucket) {
+    /** @typedef {string[]} */
+    const objects = [];
+    /**
+     * Fetch S3 objects using the given pagination token.
+     * @param {string} bucket the S3 bucket
+     * @param {string} continuationToken the pagination token
+     * @returns {Promise<string[]>} the keys of all objects
+     */
+    async function fetchObjectsWithPagination(bucket, continuationToken) {
+        const result = await s3.send(
+            new ListObjectsV2Command({
+                Bucket: bucket,
+                ContinuationToken: continuationToken
+            })
+        );
+        objects.push(...(result.Contents || []).map(object => object.Key));
+        if (result.NextContinuationToken) {
+            return fetchObjectsWithPagination(bucket, result.NextContinuationToken);
         }
     }
-    if (promises.length === 0) {
-        console.log(`No objects to delete from bucket ${event.bucket}`);
-        return;
+    await fetchObjectsWithPagination(bucket, undefined);
+    return objects;
+}
+
+/**
+ * Delete a list of keys from an S3 bucket.
+ * @param {DeleteListEvent} event the event
+ * @param {Context} context the lambda context
+ */
+async function handleDeleteList(event, context) {
+    /** @type {Promise<unknown>[]} */
+    const promises = [];
+    const chunks = splitIntoChunks(event.objects, MAX_COUNT_PER_DELETE_REQUEST);
+    console.log(`Deleting ${event.objects.length} objects in ${chunks.length}...`);
+    for (const chunk of chunks) {
+        const objectIdentifiers = chunk.map(key => { return { Key: key }; });
+        const command = new DeleteObjectsCommand({ Bucket: event.bucket, Delete: { Objects: objectIdentifiers } });
```

Comment:
Here it is "objects", afaik part of the API, right?
So we cannot rename this. 🙁

Reply:
But in the end of the line it is "identifiers" then. In some other places it's "keys". 
I don't want to spread too much confusion and also not too much additional work.
But I'd love to have the terminology a little bit unified, cleaned up and as precise as possible.